script.js

```diff
@@ -26,18 +26,98 @@ function createProductItemElement({ sku, name, image }) {
   return section;
 }
 
-function getSkuFromProductItem(item) {
-  return item.querySelector('span.item__sku').innerText;
-}
+const listOfProducts = async () => {
+  const response = await fetch('https://api.mercadolibre.com/sites/MLB/search?q=computador');
+  const { results } = await response.json();  
+  return results;
+};
+
+// function getSkuFromProductItem(item) {
+//   return item.querySelector('span.item__sku').innerText;
+// }
+
+const sumItemsPrices = async () => cartItemsList.reduce((acc, curr) => acc + curr.price, 0);
+const appendValue = async (sumReturn) => {
+  const createH2 = document.createElement('h2');
+  const selectDiv = document.getElementsByClassName('total-price')[0];
+  selectDiv.innerHTML = '';
+  createH2.innerHTML = await sumReturn;
+  selectDiv.appendChild(createH2);
+};
```

Comment:
```suggestion
const sumItemsPrices = () => cartItemsList.reduce((acc, curr) => acc + curr.price, 0);
const appendValue = (sumReturn) => {
  const createH2 = document.createElement('h2');
  const selectDiv = document.getElementsByClassName('total-price')[0];
  selectDiv.innerHTML = '';
  createH2.innerHTML = sumReturn;
  selectDiv.appendChild(createH2);
};
```
Gostei do fato de ter separado a função por sua funcionalidade deixando bem explicito o que cada uma faz, porém as duas funções não são asyncronas então você pode remover esses async e await sem nenhum problema.